sound/shairport-sync/Makefile

```diff
@@ -7,15 +7,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=shairport-sync
-PKG_VERSION:=3.3.9
-PKG_RELEASE:=2
+PKG_VERSION:=4.3.2
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/mikebrady/shairport-sync/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=17990cb2620551caa07a1c3b371889e55803071eaada04e958c356547a7e1795
+PKG_HASH:=dfb485c0603398032a00e51f84b874749bbf155b257adda3d270d5989de08bfd
 
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>, \
-		Mike Brady <mikebrady@eircom.net>
+		Mike Brady <mikebrady@eircom.net>, \
+		Ben Klang <bklang@wirehack.net>
```

Comment:
Those maintainers approved it?

Reply:
I was not sure of the etiquette here since I am upgrading the package and providing a dependency. I'll remove myself.